pallets/settlement/src/lib.rs

```diff
@@ -410,12 +420,14 @@ decl_error! {
         /// Maximum numbers of legs in a instruction > `MaxLegsInAInstruction`.
         LegsCountExceededMaxLimit,
         /// Portfolio in receipt does not match with portfolios provided by the user
-        PortfolioMismatch
+        PortfolioMismatch,
+        /// The provided settlement block number is in the past and cannot be used by the scheduler.
+        SettleOnPastBlock
     }
 }
 
 decl_storage! {
-    trait Store for Module<T: Trait> as StoCapped {
+    trait Store for Module<T: Trait> as Settlement {
```

Comment:
How this is possible ?? It was a blunder mistake If we left  `StoCapped` as the storage prefix

Reply:
😂 Good thing we're clearing settlement storage anyway in `on_runtime_upgrade`. Perhaps we should clear `StoCapped` as well.

Reply:
@maxsam4 Good idea!

Reply:
Yes, we should have clear the `StoCapped`, I am still confused about how we miss this and we haven't got any complain either from the tooling or we find out when we were doing demos. I guess it is because of the inactivity of the StoCapped pallet. 🤣

Reply:
@satyamakgec There's no problem in doing this. The only thing is that the storage location will be different for these variables than what one might expect. Unless you manually encode keys, it doesn't matter.

Reply:
This could be the matter for another PR. What do you think?

Reply:
It could be but it's just 2 loc. Can you please add
```
let sto_prefix = Twox128::hash(b"StoCapped"); 
storage::unhashed::kill_prefix(&sto_prefix);
```
to the `on_runtime_upgrade` of the settlement pallet.

Reply:
Done.